index.html

```diff
@@ -1,75 +1,420 @@
-                    Wybierz plan
-                </button>
-            </div>
-
-            <div class="header__text-under-button">
-                <div>
-                    <img src="./images/bird.svg" alt="">
-                    <a class="header__text-under-button-1">Wsparcie klienta 24/7/365</a>
-                </div>
-                <div>
-                    <img src="./images/bird.svg" alt="">
-                    <a class="header__text-under-button-2">30-dniowa gwarancja zwrotu pieniędzy</a>
-                </div>
-                <div>
-                    <img src="./images/bird.svg" alt="">
-                    <a class="header__text-under-button-3">Certyfikaty SSL w pakiecie</a>
-                </div>
-            </div>
-            <img class="header__background-img" src="./Images/background1.webp" alt="">
-
-        </div>
-
-
-
-    </header>
-</body>
-
-</html>
\ No newline at end of file
+				<div class="header__text-under-button">
+					<div>
+						<img src="./images/bird.svg" alt="" />
+						<a class="header__text-under-button-1"
+							>Wsparcie klienta 24/7/365</a
+						>
+					</div>
+					<div>
+						<img src="./images/bird.svg" alt="" />
+						<a class="header__text-under-button-2"
+							>30-dniowa gwarancja zwrotu pieniędzy</a
+						>
+					</div>
+					<div>
+						<img src="./images/bird.svg" alt="" />
+						<a class="header__text-under-button-3"
+							>Certyfikaty SSL w pakiecie</a
+						>
+					</div>
+				</div>
+				<img
+					class="header__background-img"
+					src="./Images/background1.webp"
+					alt=""
+				/>
+			</div>
+		</header>
+		<main>
+			<section class="hosting">
+				<h3 class="hosting__title-1">Wybierz swój plan hostingowy</h3>
+				<container class="hosting-plan-1">
+					<div class="hosting-plan-1__table">
+						<div class="hosting-plan-1__column-1">
+							<strong class="hosting-plan-1__text-top-1"
+								>Hosting Single</strong
+							>
+							<p class="hosting-plan-1__text-top-2">
+								Idealne rozwiązanie dla początkujących
+							</p>
+							<div class="hosting-plan-1__price">
+								<p class="hosting-plan-1__price-text">
+									25,99 zł
+								</p>
+								<div>
+									<button class="hosting-plan-1__button-1">
+										<strong>Rabat 85%</strong>
+									</button>
+								</div>
+							</div>
+							<div class="hosting-plan-1__price-primary">
+								<p class="hosting-plan-1__price-primary-1">
+									3,99
+								</p>
+								<p class="hosting-plan-1__price-primary-2">
+									zł
+								</p>
+								<p class="hosting-plan-1__price-primary-3">
+									/mies.
+								</p>
+							</div>
+							<div class="hosting-plan-1__button-2-location">
+								<button class="hosting-plan-1__button-2">
+									Dodaj do koszyka
+								</button>
+							</div>
+							<p class="hosting-plan-1__price-reneval">
+								8,99 zł/mies. po odnowieniu
+							</p>
+							<div class="hosting-plan-1__advantages">
```

Comment:
Hmm, moze lista `ul`? To totalnie wyglada jak lista

Reply:
Za duzo roboty. Zostawie divy